lib/registries/addon/routes.ts

```diff
@@ -4,6 +4,19 @@ export default buildRoutes(function() {
     this.route('index', { path: '/registries' });
     this.route('discover', { path: '/registries/discover' });
 
+    this.route('start', { path: '/registries/start' });
+
+    this.route('forms', { path: '/registries/forms' }, function() {
+        this.route('help', { path: '/help' });
+    });
+
+    this.route('drafts', { path: '/registries/drafts' }, function() {
+        this.route('draft', { path: '/:id' }, function() {
+            this.route('form');
```

Comment:
Given the updated plan, I think we want to move the "static metadata" page (previously at `.index`) to a subroute here with its own waffle. Maybe like...

Route: `registries.drafts.draft.metadata`
Waffle: `ember_registries_submission_metadata`

Mind adding that with another placeholder real quick?